testsuite/tests/toolchain/custom_install_dir/test.py

```diff
@@ -1,44 +1,53 @@
 """
 Check folders when installing binary compiler crates at a custom location
+through `alr get`
```

Comment:
Why not `alr install --prefix` here?

Reply:
Just because it is the closest to what the test was doing. We could add a new test that checks a similar idea but with `alr install`.

Reply:
`alr toolchain --install` was  introduced for people that wanted to get a toolchain from the Alire index but not use it in the Alire context. 

I feel like `alr install` is a better option for this use case.